src/units.cpp

```diff
@@ -477,7 +477,7 @@ void createUnitMap(const UnitsPtr &units, std::map<std::string, double> &unitMap
 
     } else if (isStandardUnitName(units->name())) {
         auto unit = standardUnitsList.find(units->name());
-        for (auto u : unit->second) {
+        for (auto const &u : unit->second) {
```

Comment:
Also `auto const` and `const auto` are the same, we use the latter in libCellML. So, I would change `auto const &u` to `const auto &u` (same below).